coalib/parsing/FilterHelper.py

```diff
@@ -33,3 +33,20 @@ def get_filtered_bears(cls, filter, args, all_bears=None):
         if not args or len(args) == 0:
             return all_bears
         return cls.available_filters[filter](all_bears, args)
+
+    @classmethod
+    def filter_bears(cls, bears, args):
```

Comment:
I kinda like `apply_xxx_filter`. We could shorten that even further to `apply_filter` and `apply_filters`.